pkg/helper/config_controller_helper.go

```diff
@@ -0,0 +1,13 @@
+package helper
```

Comment:
Why do we still keep config_controller? Did not we obsolete it?

Reply:
We did. Missed it